participant/models.py

```diff
@@ -85,19 +89,28 @@ def phishing_email_behaviour_and_actions(self, email_id):
         """
 
         def participant_behaviour(actions):
-            pb = ParticipantBehaviour.NEUTRAL
-            if [a for a in actions if a.get('action_type') in POSITIVE_ACTIONS]:
-                pb = ParticipantBehaviour.POSITIVE
-            if [a for a in actions if a.get('action_type') in NEGATIVE_ACTIONS]:
-                pb = ParticipantBehaviour.NEGATIVE
+            behaviour = ParticipantBehaviour.NEUTRAL
+            action_id = None
+
+            for a in actions:
```

Comment:
let's use items() here instead
```python
for action_id, action_details in actions.items():
    ... 
```

Reply:
It is condition based. So did the otherwise.